index.html

```diff
@@ -54,158 +57,76 @@ <h2 class="support-text">I’am a Software Developer</h2>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem3">
                 <h4 class="gitem3-h">Website Protfolio</h4>
                 <p class="gitem3-p">A daily selection of privately personalized
                 reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <ul class="gitem3-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <ul class="gitem3-s">
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem4">
                 <h4 class="gitem4-h">Profesional Art Printing Data More</h4>
                 <p class="gitem4-p">A daily selection of privately personalized reads;
                 no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="gitem4-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem5">
                 <h4 class="gitem5-h">Data Dashboard Healthcare</h4>
                 <p class="gitem5-p">A daily selection of privately personalized
                 reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="gitem5-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem6">
                 <h4 class="gitem6-h">Website Protfolio</h4>
                 <p class="gitem6-p">A daily selection of privately personalized
                 reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="gitem6-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
         </div>
         <div class="grid-container2">
```

Comment:
- [ ] [OPTIONAL] The project section should be created dynamically using JavaScript, kindly implement this so your project will be complete